test/Test/Sync.hs

```diff
@@ -0,0 +1,41 @@
+module Test.Sync
+       ( syncSpecs
+       ) where
+
+import IW.Core.Repo (RepoOwner (..), RepoName (..))
+import IW.Sync.Search (parseIssueUserData)
+import Test.Hspec (Spec, describe, it, shouldBe)
+
+import qualified GitHub  
+
+
+testGitHubIssueUrl :: GitHub.URL
+testGitHubIssueUrl = GitHub.URL "https://api.github.com/repos/owner123/repo123/issues/1"
+
+testBadGitHubIssueUrl1 :: GitHub.URL
+testBadGitHubIssueUrl1 = GitHub.URL "https://api.github.com/repos/owner123/issues/1"
+
+testBadGitHubIssueUrl2 :: GitHub.URL
+testBadGitHubIssueUrl2 = GitHub.URL "https://api.github.com/randomWord/owner123/repo123/issues/1"
+
+testBadGitHubIssueUrl3 :: GitHub.URL
+testBadGitHubIssueUrl3 = GitHub.URL "api.github.com/repos/owner123/repo123/issues/1"
```

Comment:
Usually we put test data at the end of the file, so that exported specs are shown immediately. Structure of the files should follow `where`-style of definitions.